x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/components/header.tsx

```diff
@@ -9,55 +9,40 @@ import React from 'react';
 
 import { useActions } from 'kea';
 
-import {
-  EuiPageHeader,
-  EuiPageHeaderSection,
-  EuiTitle,
-  EuiButton,
-  EuiButtonProps,
-  EuiLinkProps,
-} from '@elastic/eui';
-import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiPageHeader, EuiButton } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
 
 import { getAppSearchUrl } from '../../../../shared/enterprise_search_url';
 import { TelemetryLogic } from '../../../../shared/telemetry';
 
 export const EnginesOverviewHeader: React.FC = () => {
   const { sendAppSearchTelemetry } = useActions(TelemetryLogic);
 
-  const buttonProps = {
-    fill: true,
-    iconType: 'popout',
-    'data-test-subj': 'launchButton',
-    href: getAppSearchUrl(),
-    target: '_blank',
-    onClick: () =>
-      sendAppSearchTelemetry({
-        action: 'clicked',
-        metric: 'header_launch_button',
-      }),
-  } as EuiButtonProps & EuiLinkProps;
-
   return (
-    <EuiPageHeader>
-      <EuiPageHeaderSection>
-        <EuiTitle size="l">
-          <h1>
-            <FormattedMessage
-              id="xpack.enterpriseSearch.appSearch.enginesOverview.title"
-              defaultMessage="Engines Overview"
-            />
-          </h1>
-        </EuiTitle>
-      </EuiPageHeaderSection>
-      <EuiPageHeaderSection>
-        <EuiButton {...buttonProps}>
-          <FormattedMessage
-            id="xpack.enterpriseSearch.appSearch.productCta"
-            defaultMessage="Launch App Search"
-          />
-        </EuiButton>
-      </EuiPageHeaderSection>
-    </EuiPageHeader>
+    <EuiPageHeader
+      pageTitle={i18n.translate('xpack.enterpriseSearch.appSearch.enginesOverview.title', {
+        defaultMessage: 'Engines overview',
+      })}
+      rightSideItems={[
+        // eslint-disable-next-line @elastic/eui/href-or-on-click
```

Comment:
Huh, it's interesting to me that this is a lint rule at all.  Mixing an href and an onClick is valid and often times useful.

https://github.com/elastic/eui/tree/master/packages/eslint-plugin

> <EuiButton /> should either be a button or a link, for a11y purposes. When given an href the button behaves as a link, otherwise an onClick handler is expected and it will behave as a button.
>
>In some cases it makes sense to disable this rule locally, such as when cmd+click should open the link in a new tab, but a standard click should use the history.pushState() API to change the URL without triggering a full page load.

At least they acknowledge that there are valid use cases for this and it's expected to disabled this on occasion. So it sounds like this rule is in place because they want us to stop and think about the accessibility implications before proceeding with this pattern, or catch accidental usage.

Reply:
Excellent summary!! Thank you Jason!